src/wait-for-and-setup-database.php

```diff
@@ -55,12 +55,14 @@
 $connection->getSchemaBuilder()->dropIfExists('domain_messages_uuid');
 $connection->statement(<<<SQL
 CREATE TABLE IF NOT EXISTS `domain_messages_uuid` (
+  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
   `event_id` BINARY(16) NOT NULL,
   `aggregate_root_id` BINARY(16) NOT NULL,
   `version` int(20) unsigned NULL,
   `payload` varchar(16001) NOT NULL,
   PRIMARY KEY (`event_id`),
   KEY (`aggregate_root_id`),
+  KEY `pagination` (`id` ASC),
```

Comment:
Is this needed, or only used to have a key specifically named `pagination`? I was under the impression the `AUTO_INCREMENT` line above would provide an index already. Otherwise, is this to ensure the direction (`ASC`) that would otherwise not be guaranteed to be sorted the right way?

I have a lot of questions here, and not because I think anything is wrong. I think this might be something I've not known about in the past! I sorta take for granted that defining `id` like this will give me a unique index and always assumed the default sort order would be fine for cases like this but maybe not?

Reply:
Normally the primary key automatically creates the index, this is "just" an incremental id, not the primary key, therefor it doesn't always have an index.